qt/app.py

```diff
@@ -345,10 +345,9 @@ def __init__(self, config, appInstance, qapp):
         # force settingdialog if it is not configured
         if not config.isConfigured():
             message = _(
-                '{appName} is not configured. Would you like '
-                'to restore a previous configuration?') \
-                .format(appName=self.config.APP_NAME)
-
+                '{app_name} appears to be running for the first time as no '
+                'configuration is found. Restore a previous configuration?') \
+                .format(app_name=self.config.APP_NAME)
```

Comment:
I think new users (and possibly also users who re-install BiT on another computer or even going to restore) may need a hint
- where to find a existing configuration file (in the snapshots or also manually copied from an old installation)
- when to use which configuration file (data restore = from most recent snapshot, migration to new computer = copy config file from old computer, first time = no need to restore a prev configuration but the after clicking on `OK` the next dialog allows to capture a new configuration)
- the word "restore" is confusing IMHO because it is also used in the context auf "backups" to restore data but here we restore meta data (a config file)

How about this modified message:

    ...
    {app_name} appears to be running for the first time as no
    configuration file can be found.
    
    Do you want to import an existing configuration from a backup target folder or another computer?

If `Yes` is clicked the next dialog should show where to find configuration files (eg. mention `backintime --diagnostics` to find the config file folder for an existing installation on another computer).

Reply:
I thought the same but was lazy. :D OK, I'll work on it.

Reply:
OK, closing this conversation as out dated.